### PR TITLE
Adding NTT to the list

### PR DESCRIPTION
**Pull request by CsatariGergely**

### CODE REVIEW

File: src/config/featured.json [hunk 1]
```diff
@@ -14,6 +14,13 @@
       "description": "KNIX MicroFunctions is a serverless computing platform that combines container-based resource isolation with a lightweight execution model using processes to significantly improve resource efficiency and decrease the function startup latency. KNIX MicroFunctions works in Knative as well as bare metal or virtual machine-based environments.",
       "logo": "https://raw.githubusercontent.com/knix-microfunctions/knix/develop/logo/KNIX_logo_stack.jpg"
     },
+    {
+      "name": "NTT",
+      "repo_url": "https://github.com/nokia/ntt",
+      "homepage": null,
```
Comment (5nord) [inline src/config/featured.json:20]:
```suggestion
      "homepage": "https://nokia.github.io/ntt/",
```

File: src/config/featured.json [hunk 2]
```diff
@@ -14,6 +14,13 @@
       "description": "KNIX MicroFunctions is a serverless computing platform that combines container-based resource isolation with a lightweight execution model using processes to significantly improve resource efficiency and decrease the function startup latency. KNIX MicroFunctions works in Knative as well as bare metal or virtual machine-based environments.",
       "logo": "https://raw.githubusercontent.com/knix-microfunctions/knix/develop/logo/KNIX_logo_stack.jpg"
     },
+    {
+      "name": "NTT",
```
Comment (5nord) [inline src/config/featured.json:18]: We are still not 100% sure if it's "NTT" or "ntt", but the trend goes:
```suggestion
      "name": "ntt",
```

File: src/config/featured.json [hunk 3]
```diff
@@ -14,6 +14,13 @@
...
+      "repo_url": "https://github.com/nokia/ntt",
+      "homepage": null,
+      "description": "Modern tools for TTCN-3. ntt gives you the tools you need to write tests efficiently.",
+      "logo": null
```
Comment (5nord) [inline src/config/featured.json:22]: The least ugly we have ;)
```suggestion
      "logo": "https://nokia.github.io/ntt/static/ntt-mono-blue.png"
```

**Review by 5nord — APPROVED** [review]
Thanks!

File: src/config/featured.json (hunk 1 above)
Comment (CsatariGergely) [reply src/config/featured.json:20]: Repo url will be already a clickable link.

Comment (5nord) [reply src/config/featured.json:20]: This is not the Repo, but the homepage. Would be nice if we'd link this.

Comment (CsatariGergely) [reply src/config/featured.json:20]: Ah okay, now I realized, that this is different from the repo link.

**Review by 5nord — APPROVED** [review]
Super

**Pull request merged**